rrrspec-client/lib/rrrspec/client/support.rb

```diff
@@ -40,7 +40,26 @@ def start_taskset(conf, rsync_name)
         return taskset
       end
 
-      def show_result(taskset, verbose=false)
+      def show_result(taskset, verbose=false, show_errors=false)
+        if show_errors
```

Comment:
@draftcode yes, we run it in console. Console automatically scrolls down to the very end of the output and the very first thing that we want to see with ```--show-errors``` option is [summary](https://github.com/sauliusgrigaitis/rrrspec/blob/show-errors/rrrspec-client/lib/rrrspec/client/support.rb#L66) . If we see that there are some errors in the [summary](https://github.com/sauliusgrigaitis/rrrspec/blob/show-errors/rrrspec-client/lib/rrrspec/client/support.rb#L66), then we just scroll up to investigate which tests failed. I also think that output of ```--verbose``` should be above [summary](https://github.com/sauliusgrigaitis/rrrspec/blob/show-errors/rrrspec-client/lib/rrrspec/client/support.rb#L66), but that's another topic.